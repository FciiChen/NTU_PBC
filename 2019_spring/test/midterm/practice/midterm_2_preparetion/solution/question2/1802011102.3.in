7FVSBTP7, FWD7dhTvKg,QCIhBq,Sksnodsi,4H6Z7,DUF,IWEzAoR,btu6voGWj,DyZlM97JDD,GuMkOzFz
ZdBBxB, dKkoD3,wfXGZWMWf,m8HanabL2,bLI,ZopwPMmmE,KEp,VWW,DwAz,cyTOW3XQ5W
HBuGE41aR, GXVCiO1D,99Bnklli8,q7hL90,jbfQZjIA,xj4G,WixosJq,ICN9A95Ts,PecfnOpCa,D6hUr
3YF8uj26ft, SZnpV7,Gf4W9,9aFtpRieB0,fJiL1,VUdwhGQn,8Af8nqYf,anaJUK,nk70p6,WTz0j5tZvc
BREAK
7FVSBTP7 ,  nvoBCp
HBuGE41aR ,  6CjyjzQBN
3YF8uj26ft ,  iknSzg
ZdBBxB ,  qScm00ZuH
BREAK
