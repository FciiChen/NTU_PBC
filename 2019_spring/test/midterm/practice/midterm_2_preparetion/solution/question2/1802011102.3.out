7FVSBTP7   FWD7dhTvKg QCIhBq     Sksnodsi   4H6Z7      DUF        IWEzAoR    btu6voGWj  DyZlM97JDD GuMkOzFz   nvoBCp    
ZdBBxB     dKkoD3     wfXGZWMWf  m8HanabL2  bLI        ZopwPMmmE  KEp        VWW        DwAz       cyTOW3XQ5W qScm00ZuH 
HBuGE41aR  GXVCiO1D   99Bnklli8  q7hL90     jbfQZjIA   xj4G       WixosJq    ICN9A95Ts  PecfnOpCa  D6hUr      6CjyjzQBN 
3YF8uj26ft SZnpV7     Gf4W9      9aFtpRieB0 fJiL1      VUdwhGQn   8Af8nqYf   anaJUK     nk70p6     WTz0j5tZvc iknSzg    
